src/test/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilderTest.java

```diff
@@ -301,6 +301,7 @@ private void validatePod(Pod pod, boolean directConnection) {
 
     private void validatePod(Pod pod, boolean fromYaml, boolean directConnection) {
         assertThat(pod.getMetadata().getLabels(), hasEntry("some-label", "some-label-value"));
+        assertEquals("Never", pod.getSpec().getRestartPolicy());
```

Comment:
https://github.com/jenkinsci/kubernetes-plugin/pull/906/checks?check_run_id=1473530018

Reply:
yeah, looking into it.